pkg/package/service/config.go

```diff
@@ -58,9 +82,12 @@ func (ps packageService) ExportPackageToTemporaryConfig(pkg domain.Package) (str
 		return "", fmt.Errorf("create temporary config file: %v", err)
 	}
 	defer configFile.Close()
+  
+  return configFile.Name(), exportPackageToWriter(pkg, conf)
```

Comment:
Looking at the failed test it seems like the last `conf` doesn't exist and should really be `configFile.Name()`.